From fc45beed40a86c8fc38fc430765c1c5d2c9961c2 Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Thu, 4 Mar 2021 09:26:48 -0300
Subject: [PATCH] setup.py: Support idna version 3.1
Organization: O.S. Systems Software LTDA.

Change idna required version to use python3-idna recipe that currently
is on version 3.1.

Without this change python-request doesn't work as the idna version
doesn't match the required versions. When bump python-request recipe,
verify if the upstream project bumped the idna version and drop this
patch.

As this recipe was tested with python3-idna_3.1 don't allow higher
versions.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7ba4b2a2..9d3b973e 100755
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,7 @@ packages = ['requests']
 
 requires = [
     'chardet>=3.0.2,<5',
-    'idna>=2.5,<3',
+    'idna>=2.5,<3.2',
     'urllib3>=1.21.1,<1.27',
     'certifi>=2017.4.17'
 
-- 
2.30.1

